src/umpire/CMakeLists.txt

```diff
@@ -217,6 +240,10 @@ install(FILES
   ${umpire_headers}
   DESTINATION include/umpire)
 
+install(FILES
```

Comment:
When doing the umpire_interface cmake changes (see PR #631 ), I had to add a `OBJECTS DESTINATION` line for the install line (something like this https://github.com/LLNL/Umpire/blob/020372220fd6e59c1e085f62301b296d7170e344/src/umpire/CMakeLists.txt#L235) - is that needed here too, or is this good enough?

Reply:
As discussed, no object install is required.

Reply:
Update: not needed